### ISSUE TITLE
Core Dump when running expamles

### ISSUE THREAD

**Issue by LaFeuilleMorte** [VAST-AI-Research/TripoSR#14]
2024-03-05 17:28:53,505 - INFO - Initializing model finished in 16985.25ms.
2024-03-05 17:28:53,505 - INFO - Processing images ...
2024-03-05 17:28:54,974 - INFO - Processing images finished in 1468.80ms.
2024-03-05 17:28:54,974 - INFO - Running image 1/1 ...
2024-03-05 17:28:54,974 - INFO - Running model ...
Segmentation fault (core dumped)

**Comment by bennyguo** [comment]
Hi! Are you using CPU for inference?

**Comment by LaFeuilleMorte** [comment]
No, I print the device which the model will be sent, it's GPU. My computer is RTX 3090 + torch 2.2.1 + cuda 11.8
![image](https://github.com/VAST-AI-Research/TripoSR/assets/43473814/23584423-64bf-4b0c-bb98-754f948bfc07)
